File: verification-packs/polyglot-30-route-formal-equivalence-v1/evidence/routes/java-to-rust/certification/artifacts/real-repository/source-runtime/RouteHarness.java
public final class RouteHarness {
    public static void main(String[] args) {
        var actual0 = Difference.difference(20, 7);
        if (actual0 != 13) throw new AssertionError("case 0");
        System.out.println("ELMOS_OBSERVATION\t0\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual0).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
        var actual1 = Difference.difference(3, 8);
        if (actual1 != 0) throw new AssertionError("case 1");
        System.out.println("ELMOS_OBSERVATION\t1\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual1).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
        var actual2 = Difference.difference(4, 4);
        if (actual2 != 0) throw new AssertionError("case 2");
        System.out.println("ELMOS_OBSERVATION\t2\tb64\t" + java.util.Base64.getEncoder().encodeToString(String.valueOf(actual2).getBytes(java.nio.charset.StandardCharsets.UTF_8)));
    }
}

File: verification-packs/polyglot-30-route-formal-equivalence-v1/evidence/routes/typescript-to-go/certification/artifacts/real-repository/route_harness.go
package main

import (
    "encoding/base64"
    "fmt"
)

func main() {
    actual0 := difference(20, 7)
    if actual0 != 13 { panic("case 0") }
    fmt.Printf("ELMOS_OBSERVATION\t0\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual0))))
    actual1 := difference(3, 8)
    if actual1 != 0 { panic("case 1") }
    fmt.Printf("ELMOS_OBSERVATION\t1\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual1))))
    actual2 := difference(4, 4)
    if actual2 != 0 { panic("case 2") }
    fmt.Printf("ELMOS_OBSERVATION\t2\tb64\t%s\n", base64.StdEncoding.EncodeToString([]byte(fmt.Sprint(actual2))))
}
